src/marketplaces/ae.ts

```diff
@@ -34,5 +39,6 @@ export const AE = new AmazonMarketplace({
   sellingPartner: {
     region: sellingPartnerRegions.EU,
     sellerCentralAuthUri: sellerCentralAuthUris.NA,
+    vendorCentralAuthUri: vendorCentralAuthUriTemporary, // This will be generated from Vendor Central URI
```

Comment:
Not a big fan of the `vendorCentralAuthUriTemporary` approach. I understand the constraint, but we should figure out a better way. We can do this as a separate PR though.

Reply:
I opened a new issue for this: https://github.com/ScaleLeap/amazon-marketplaces/issues/403
Will think about them. Thanks.